docs/using-carbondata-to-visualization_in-notebook.md

```diff
@@ -0,0 +1,80 @@
+## Preparing
+It should already installed docker environment before using carbondata in notebook.
+
+## Downloading docker images of carbondata notebook 
+
+Downloading latest version of docker images of carbondata notebook 
+
+```shell
+docker pull xubo245/all-carbondata-notebook:latest
```

Comment:
this is your personal folder,   is it working for all users?

Reply:
it's not personal folder, it's public images address of docker hub,  all user can download the carbondata images by this command if they can access docker hub official website